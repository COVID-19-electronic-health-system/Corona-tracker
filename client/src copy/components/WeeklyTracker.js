import React, { Component } from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import WeeklyTrackerDay from './WeeklyTrackerDay';
class WeeklyTracker extends Component {
    constructor(props) {
        super(props);
        this.state = {
            //example of data that will be passed to it 
            dates: [
                {
                    "date": "2020-03-11T14:00:00-05:00",
                    "status": "10"
                },
                {
                    "date": "2020-03-12T15:00:00-05:00",
                    "status": "10"
                }, 
                {
                    "date": "2020-03-13T13:30:00-05:00",
                    "status": "09"
                }
            ]
        };

    }
    render() {
        return (
            <WeeklyTrackerDay/>
        );
    }
}

export default WeeklyTracker;