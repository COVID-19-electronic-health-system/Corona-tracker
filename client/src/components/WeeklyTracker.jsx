import React from 'react';
import WeeklyTrackerDay from './WeeklyTrackerDay';

const WeeklyTracker = () => {
  // constructor(props) {
  //   super(props);
  //   this.state = {
  //     dates: [
  //       {
  //         date: '2020-03-11T14:00:00-05:00',
  //         status: '10',
  //       },
  //       {
  //         date: '2020-03-12T15:00:00-05:00',
  //         status: '10',
  //       },
  //       {
  //         date: '2020-03-13T13:30:00-05:00',
  //         status: '09',
  //       },
  //     ],
  //   };
  // }

  return <WeeklyTrackerDay />;
};

export default WeeklyTracker;
