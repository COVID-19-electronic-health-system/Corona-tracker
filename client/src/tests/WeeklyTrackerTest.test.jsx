import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import WeeklyTracker from '../components/WeeklyTracker';

const mockStore = configureMockStore();

describe('WeeklyTracker', () => {
  it('should render a WeeklyTracker component ', () => {
    const store = mockStore();
    window.HTMLCanvasElement.prototype.getContext = () => {};

    const wrapper = mount(
      <Provider store={store}>
        <WeeklyTracker chartType="test" />
      </Provider>
    );
    expect(wrapper.find('WeeklyTracker')).toMatchSnapshot();
  });
});
