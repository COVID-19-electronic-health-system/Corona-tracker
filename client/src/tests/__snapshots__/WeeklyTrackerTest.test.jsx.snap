// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeeklyTracker should render a WeeklyTracker component  1`] = `ReactWrapper {}`;
